crates/trigger-http/src/wasi.rs

```diff
@@ -98,27 +98,27 @@ impl HttpExecutor for WasiHttpExecutor {
                 }
             };
 
-        let span = tracing::debug_span!("execute_wasi");
+        // TODO(Caleb): Verify that using tracing::Span::current() does not cause regressions
         let handle = task::spawn(
             async move {
                 let result = match handler {
                     Handler::Latest(proxy) => {
                         proxy
                             .wasi_http_incoming_handler()
                             .call_handle(&mut store, request, response)
-                            .instrument(span)
+                            .instrument(tracing::Span::current())
```

Comment:
```suggestion
                            .in_current_span()
```

:shrug:

Reply:
Actually I wonder if these are actually doing anything? There is a `in_current_span` on the async block already.